src/components/Data/Collections/CreateOrUpdate.vue

```diff
@@ -86,18 +86,22 @@
 
           <!-- Actions -->
           <div class="row">
-            <div class="col s6">
+            <div class="col s5 m4 l4">
                 <a tabindex="6" class="btn-flat waves-effect" @click.prevent="cancel">Cancel</a>
                 <button type="submit" class="btn primary waves-effect waves-light">
                   <i v-if="!collectionName" class="fa fa-plus-circle left"></i>
                   <i v-else class="fa fa-pencil left"></i>
                   {{collectionName ? 'Update' : 'Create'}}
                 </button>
-                <p class="error">{{error}}</p>
+            </div>
+            <div class="col s7 m8 l8" v-if="error">
+              <div class="card error red white-text">
+                <i class="fa fa-times dismiss-error" @click="dismissError()"></i>
+                An error occurred while {{collectionName ? 'updating' : 'creating'}} collection: <br>{{error}}
```

Comment:
I've seen you set HTML in the error string so shouldn't this be wrapped with 3 brackets ?